util/bidderutil/bidderutil.go

```diff
@@ -0,0 +1,27 @@
+package bidderutil
```

Comment:
few thoughts here,

- File name should be in snake case i.e `bidder_util.go`
- May be no need for creating separate package for this util. This can be added `adapters` package similar to other utils we have today.
  ```
      adpapters/openrtb_util.go
       adpapters/bidder_util.go
  ```
- also should we rename `bidder_util.go` to `response_util.go`

Reply:
> * File name should be in snake case i.e `bidder_util.go`
I am being consistent with what we have in utils package.
> * May be no need to create separate package for this util. This can be added  in `adapters` package similar to other utils we have today.
There is already a utils folder under adapters. So this should be fine. I have changed the location to httputil package instead of creating a new one.
>   ```
>       adpapters/openrtb_util.go
>       adpapters/bidder_util.go
>   ```
> * also should we rename `bidder_util.go` to `response_util.go`
renamed to responseutil.go in consistent with existing file names.
[87d05e0](https://github.com/prebid/prebid-server/pull/2638/commits/87d05e0af3b5db96bcbc7e269be6d79a3c99a522)